build/assets/custom-dictionary.txt

```diff
@@ -1,49 +1,672 @@
-personal_ws-1.1 en 1315 utf-8
+ABCpred
```

Comment:
Do you want to sort in a case insensitive manner? I'm pretty sure @cgreene has a one-liner for dictionary updates that he shared in a different Manubot repository.

Reply:
https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/.github/PULL_REQUEST_TEMPLATE.md#spellcheck-step